docker/templates/base/latest/Dockerfile

```diff
@@ -14,55 +16,52 @@ RUN apt-get update && \
       telnet \
       sudo && \
     rm -rf /var/lib/apt/lists/*
+ENV DEBIAN_FRONTEND teletype
 
 # Configure default JAVA_HOME path
-RUN ln -s java-7-openjdk-amd64 /usr/lib/jvm/default-java
+RUN ln -s java-8-openjdk-amd64 /usr/lib/jvm/default-java
 
 # Install JRuby (the Ubuntu JRuby package is 1.5.6!)
 RUN mkdir -p /var/lib/jruby \
-    && curl -SL http://jruby.org.s3.amazonaws.com/downloads/1.7.20/jruby-bin-1.7.20.tar.gz \
+    && curl -SL http://jruby.org.s3.amazonaws.com/downloads/1.7.26/jruby-bin-1.7.26.tar.gz \
     | tar -z -x --strip-components=1 -C /var/lib/jruby
 ENV PATH /var/lib/jruby/bin:$PATH
 
-ENV TOMCAT_HOME /var/lib/tomcat7
-ENV TOMCAT_CONFIG /etc/tomcat7
-
-ENV KILLBILL_CONFIG /etc/killbill
+ENV TOMCAT_HOME=/var/lib/tomcat7 \
+    TOMCAT_CONFIG=/etc/tomcat7 \
+    KILLBILL_CONFIG=/etc/killbill
 
 # Configure Tomcat
-RUN mkdir -p /usr/share/tomcat7/common/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/common/classes
-RUN mkdir -p /usr/share/tomcat7/common && chown -R tomcat7:tomcat7 /usr/share/tomcat7/common
-RUN mkdir -p /usr/share/tomcat7/server/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/server/classes
-RUN mkdir -p /usr/share/tomcat7/server && chown -R tomcat7:tomcat7 /usr/share/tomcat7/server
-RUN mkdir -p /usr/share/tomcat7/shared/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/shared/classes
-RUN mkdir -p /usr/share/tomcat7/shared && chown -R tomcat7:tomcat7 /usr/share/tomcat7/shared
-RUN mkdir -p /tmp/tomcat7-tomcat7-tmp && chown -R tomcat7:tomcat7 /tmp/tomcat7-tomcat7-tmp
-RUN chmod g+w /etc/tomcat7/catalina.properties
-RUN rm -rf $TOMCAT_HOME/webapps/*
-
-RUN mkdir -p $KILLBILL_CONFIG
-RUN chown -R tomcat7:tomcat7 $KILLBILL_CONFIG
+RUN mkdir -p /usr/share/tomcat7/common/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/common/classes && \
+    mkdir -p /usr/share/tomcat7/common && chown -R tomcat7:tomcat7 /usr/share/tomcat7/common && \
+    mkdir -p /usr/share/tomcat7/server/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/server/classes && \
+    mkdir -p /usr/share/tomcat7/server && chown -R tomcat7:tomcat7 /usr/share/tomcat7/server && \
+    mkdir -p /usr/share/tomcat7/shared/classes && chown -R tomcat7:tomcat7 /usr/share/tomcat7/shared/classes && \
+    mkdir -p /usr/share/tomcat7/shared && chown -R tomcat7:tomcat7 /usr/share/tomcat7/shared && \
+    mkdir -p /tmp/tomcat7-tomcat7-tmp && chown -R tomcat7:tomcat7 /tmp/tomcat7-tomcat7-tmp && \
+    chmod g+w /etc/tomcat7/catalina.properties && \
+    rm -rf $TOMCAT_HOME/webapps/* && \
+    mkdir -p $KILLBILL_CONFIG && \
```

Comment:
Maybe these two should be a different `RUN` command, since it's not really part of the tomcat cleanup?